pages/common/opt.md

```diff
@@ -0,0 +1,15 @@
+# opt
+
+> LLVM .bc -> .bc modular optimizer and analysis printer. `opt` takes LLVM source files as input, runs the specified optimizations or analyses on it, and then outputs the optimized file or the analysis results.
+
+- Run an optimization or an analysis in a bitcode file:
+
+`opt {{-passname}} {{file.bc}} -S -o {{file_opt.bc}}`
+
+- Print the Control Flow Graph of a function to a "dot" file:
```

Comment:
Perhaps `Output` would be a better term than `Print` here if it's going to a file and not a terminal?

Also, can `file.bc` be `path/to/file.bc` please?